features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/fancy/templates/api/documentation/download.jag

```diff
@@ -0,0 +1,20 @@
+<%
+include("/jagg/jagg.jag");
+var store = jagg.module("manager").getAPIStoreObj();
+var domain=request.getParameter("tenant");
+var resourceUrl = request.getParameter("resourceUrl");
+
+    if(publisher.getDocument instanceof Function){
```

Comment:
Do we really need this check? I think we can remove it since the function will always exist in the StoreHostObject.